backend/src/app.module.ts

```diff
@@ -17,6 +17,8 @@ const __dirname = path.dirname(__filename);
 
 config({ path: './.env' });
 
+const enableIamUserOnboard = process.env.ENABLE_IAM_USER_ONBOARD === 'true';
+
 const portalOptions: PortalModuleOptions = {
```

Comment:
`const authCallbackProvider = process.env.ENABLE_IAM_USER_ONBOARD ? AuthCallbackProvider :  null;`

I think this should be enough and then just 
{
...
authCallbackProvider,
...}